core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -200,118 +200,6 @@ public void testScenario_111() throws Exception {
    }
 
    /**
-    * ISPN-3315: the remote get is trigger in stable state and the reply received after the rebalance_start command.
-    * However, the old owner will receive the request after the state transfer and he no longer has the key.
```

Comment:
GC would be the most common one. But when the cluster is under a lot of load, some nodes may also discard the `CacheTopologyControlCommand` message and have to wait for a retransmission.